package.json

```diff
@@ -5,9 +5,13 @@
   "description": "Extenda Retail Technology Radar",
   "main": "src/js/index.jsx",
   "scripts": {
-    "build": "webpack-cli --mode production",
+    "build": "npm run build:frontend && npm run build:backend",
+    "build:frontend": "webpack-cli --mode production",
+    "build:backend": "npm run build --prefix server",
     "lint:js": "eslint src test",
     "start": "webpack-dev-server --open --watch --mode development --output-path build",
+    "backend": "PORT=3000 npm start --prefix server",
```

Comment:
This env wont be portable(Windows anybody?) so a way will be to do `npm run env PORT=3000 -- npm start --prefix server`

Reply:
Thanks, didn't know about `npm run env`!